src/super_gradients/training/losses/ohem_ce_loss.py

```diff
@@ -11,7 +11,8 @@ def __init__(self,
                  threshold: float,
                  mining_percent: float = 0.1,
                  ignore_lb: int = -100,
-                 num_pixels_exclude_ignored: bool = True):
+                 num_pixels_exclude_ignored: bool = True,
+                 criteria: torch.nn.Module = None):
```

Comment:
perhaps the type should be `_Loss` instead of `nn.Module`

Reply:
fixed